app/src/main/AndroidManifest.xml

```diff
@@ -4,8 +4,6 @@
     package="com.mapbox.maps.testapp">
 
     <uses-permission android:name="android.permission.ACCESS_FINE_LOCATION" />
-    <uses-permission android:name="android.permission.WRITE_EXTERNAL_STORAGE" />
-    <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE" />
```

Comment:
Finally! 👍

Reply:
Actually we need it for `com.squareup.leakcanary:leakcanary-android-core:2.4` checking if we could bump the dependency to fix the storage permission.

Reply:
ok, by default leakcanary doesn't try to request this permission, so I think it's fine to remove.